### PR TITLE
profile page

### PR DESCRIPTION
**Pull request by rogerbutt**

### CODE REVIEW

File: client/src/actions/auth.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import {LOGIN_USER_REQUEST, LOGIN_USER_SUCCESS, LOGIN_USER_FAILURE, LOGOUT_USER, SIGNUP_REQUEST, SIGNUP_SUCCESS, SIGNUP_FAILURE} from '../constants/ActionTypes.js';
-import {pushState} from 'redux-router';
+import {replaceState, pushState} from 'redux-router';
```
Comment (rong2) [inline client/src/actions/auth.js]: I don't see replaceState being used.

File: client/src/components/AuthenticatedComponent.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ export function requireAuthentication(Component) {
             console.log(this.props);
             if (!this.props.isAuthenticated) {
                 let redirectAfterLogin = this.props.location.pathname;
-                this.props.dispatch(pushState(null, '/login?next=${redirectAfterLogin}'));
+                this.props.dispatch(pushState(null, `/login?next=${redirectAfterLogin}`));
```
Comment (rong2) [inline client/src/components/AuthenticatedComponent.js:25]: This is a different kind of apostrophe, is that on purpose?

Comment (rogerbutt) [inline client/src/components/AuthenticatedComponent.js:25]: I think so, it puts the variable into the string without the + stuff.

File: client/src/containers/Profile.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import React, { PropTypes, Component } from 'react';
 import { connect } from 'react-redux';
+import { getUserInfo } from '../actions/user.js';
```
Comment (rong2) [inline client/src/containers/Profile.js:3]: Why the .js at the end of user?

**Pull request merged**